wallet/src/ui.rs

```diff
@@ -313,14 +313,19 @@ pub enum PrivateKey {
     Freezing(FreezerKeyPair),
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum BalanceInfo {
     /// The balance of a single asset, in a single account.
     Balance(u64),
     /// All the balances of an account, by asset type.
-    AccountBalances(HashMap<AssetCode, u64>),
+    AccountBalances((HashMap<AssetCode, u64>, HashMap<AssetCode, AssetInfo>)),
```

Comment:
Since each `AssetCode` corresponds to a balance and an `AssetInfo`, I think it's better to have just one map, i.e., `HashMap<AssetCode, (u64, AssetInfo)>` if this doesn't make the enum too complicated to construct.

Reply:
I did it this way to avoid sending the same AssetInfo struct multiple times when you get all the balances, but that's probably a non issue.  I can make this change I don't think it's any more complicated now